scripts/Engineering/gui/engineering_diffraction/tabs/fitting/data_handling/data_model.py

```diff
@@ -162,17 +167,25 @@ def get_ws_sorted_by_primary_log(self):
             ws_list = ws_list[::-1]
         return ws_list
```

Comment:
We want this function to return the workspaces only with xunit TOF (which are passed to the sequential fitting) - it looks like this will still pick up dSpacing workspaces, is that correct?

Reply:
Good spot, I'll have a look at sorting this out